docs/paper/dev/api/obsolete-api-migration/material.md

```diff
@@ -0,0 +1,72 @@
+---
+slug: /dev/material
+---
+
+# Material Migration Guide
+
+Material was originally built when every single item had a corresponding block.
+However, on modern versions of Material, this is no longer the case. 
+It’s important to understand that from now on, what was once called Material is now split into two categories: ``ItemType`` and ``BlockType``.
+
+This guide is meant to outline some of the steps you can take in your codebase in order to remove usage of the now deprecated 
+Material API. All places in the API that once used Material now only take an ItemType or a BlockType depending on the context.
+
+## ItemType
+ItemType represents a type of Item that can only be given in an inventory and may not have a physical block in the world. 
+In contexts where Material was previously used in inventories or ItemStacks, ItemType may be used instead.
+
+#### Before
+```java
+new ItemStack(Material.AIR);
+```
+#### After
+```java
+ItemStack.of(ItemType.AIR);
+```
+*Using static constructors is encouraged over the use of the built-in constructor.
```

Comment:
I think all built-in constructors are deprecated in upstream's PR now.

Reply:
Yep, will remove.